pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -264,6 +265,55 @@ var (
 
 		return result
 	}()
+
+	// managedAtServiceCreationLBServiceAnnotations maps platform to annotation keys
+	// that the operator manages, but only during service creation. This means that
+	// the operator will not apply a desired annotation update right away.
+	// Instead, it will set a Progressing=True status condition with a message explaining
+	// how to apply the change manually (e.g., by deleting the service so that it can be
+	// recreated).
+	//
+	// Note that the ingress.operator.openshift.io/auto-delete-load-balancer annotation enables
+	// these annotations to be applied immediately by automatically deleting and recreating the
+	// service.
+	managedAtServiceCreationLBServiceAnnotations = func() map[configv1.PlatformType]sets.Set[string] {
+		result := map[configv1.PlatformType]sets.Set[string]{
+			configv1.AWSPlatformType: sets.New[string](
+				// Annotation for configuring load balancer subnets.
+				// This requires service recreation because NLBs do not support updates to subnets,
+				// the operator cannot detect errors if the subnets are invalid, and to prevent
+				// compatibility issues during upgrades since this annotation was previously unmanaged.
+				awsLBSubnetsAnnotation,
+				// Annotation for configuring load balancer EIPs.
+				// This requires service recreation to prevent compatibility issues during upgrades since
+				// this annotation was previously unmanaged.
+				awsEIPAllocationsAnnotation,
+			),
+		}
+
+		// Add the annotations that do NOT support mutable scope.
+		for platform, annotation := range InternalLBAnnotations {
+			if _, ok := platformsWithMutableScope[platform]; !ok {
+				for name := range annotation {
+					if result[platform] == nil {
+						result[platform] = sets.New[string]()
+					}
+					result[platform].Insert(name)
+				}
+			}
```

Comment:
A little easier to read:
```suggestion
			if _, ok := platformsWithMutableScope[platform]; ok {
				continue
			}
			for name := range annotation {
				if result[platform] == nil {
					result[platform] = sets.New[string]()
				}
				result[platform].Insert(name)
			}
```